doc/win32-usb-troubleshoot.md

```diff
@@ -6,27 +6,66 @@ While this makes it easier to develop nRF Connect Desktop applications, it might
   - This uses the *rules* already installed for actually *binding* a driver to the newly-connected nRF USB device.
   - A yellow bubble appears from the system tray, indicating driver installation, like this: 
   ![screenshot](win32-drivers-installing.png)
 
 - A `LIBUSB_ERROR_NOT_FOUND` is shown on the log of the nRF Connect Desktop application.
-- After a couple of minutes, Windows 7 finishes binding the libusb driver to the nRF USB device.
+- After a time (that varies between a few seconds and a couple of minutes), Windows 7 finishes binding the libusb driver to the nRF USB device.
   - A yellow bubble appears from the system tray, indicating drivers are ready, like this:
   ![screenshot](win32-drivers-ready.png)
 
+Please note that a USB device is "new" for Windows if the combination of its USB vendor ID, product ID, revision, and serial number have never been seen before. In other words: changing the firmware in an nRF device, or setting it into bootloader mode will make it appear as "new" for Windows.
+
+Therefore, this error might happen the first time that an nRF USB device is connected, the first time it enters bootloader mode, and the first time it is loaded with application-specific USB-capable firmware.
+
 #### Workaround
 
 - Close all programs and processes that might be using nRF USB devices (including nRF Connect Desktop).
 - Connect the nRF USB device(s).
 - **Wait** until Windows 7 has finished binding the right drivers to the USB device(s).
 - Launch nRF Connect Desktop.
+
+If the nRF USB device enters bootloader mode, or is loaded with new firmware, you may need to repeat this workaround.
+
+## USB devices without an expected interface
+
+A `LIBUSB_ERROR_NOT_FOUND` error shows up when connecting an nRF USB device **without** a DFU trigger interface.
+
+In order to check if this is also your scenario, launch the Windows Device Manager, set the view to "Devices by Connection", and find the part of the USB device tree with the nRF USB Device.
+
+The nRF USB device shall appear as an "USB Composite Device" and, beneath that, each USB interface appears as a device in the tree. nRF Connect Desktop *expects* a "nRF Connect DFU trigger" interface to be present, like so:
+
+![screenshot](win32-usbtree-expected.png)
+
+The "nRF Connect DFU trigger" interface is implemented by the [Nordic USB DFU trigger library](http://infocenter.nordicsemi.com/topic/com.nordic.infocenter.sdk5.v15.0.0/group__nrf__dfu__trigger__usb.html). This scenario happens when the firmware loaded in the nRF USB device does not use (or misuses) this USB DFU library.
+
+#### Workaround
+
+Most nRF Connect Desktop applications will automatically load the nRF USB device with firmware known to work.
+
+- If your nRF device does *not* have a Segger IMCU:
+  - Unplug the nRF device from its USB port.
+  - Hold down the reset button.
+  - Pluf the nRF device into a USB port. One of the LEDs should be pulsing, indicating the nRF device is in DFU bootloader mode.
```

Comment:
Pluf -> plug.
I think you can safely state that the red LED should be pulsing for the bootloader.